dnf-behave-tests/dnf/dnfdaemon/repolist.feature

```diff
@@ -0,0 +1,15 @@
+@not.with_os=rhel__ge__7
+@no_installroot
+
+Feature: Test dnfdaemon-client repolist
+
+Background: Configure repositories and check if daemon is up
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-fedora-updates"
+    And dnfdaemon-server is up and running
```

Comment:
I would prefer not run dnfdaemon-server in a step - just to be able to share the scenarios with microdnf which does not need it.
I was thinking about using some decorator like `@dnfdaemon` for scenarios suitable for daemon and run the server from environment.py:before_scenario() callback.